### DISCUSSION TITLE
Torsion Database Error in AbDesign Workflow

### DISCUSSION THREAD

**Discussion by durjoy-bty** [RosettaCommons/rosetta#183] [Rosetta]
Hi everyone,
I’ve been working on an enzyme design project using the AbDesign [workflow] (https://github.com/Fleishman-Lab/AbDesign_for_enzymes/blob/master/README.md) from this repository, and I’ve encountered an issue that I’ve been unable to resolve. While running the workflow, I receive a "std::out_of_range" error, and the process aborts with a Signal 6 (SIGABRT). The error is at the **torsion database step**. 
Here’s a snippet of the error message:
terminate called after throwing an instance of 'std::out_of_range'
what():  map::at
Got some signal... It is:6
Signal 6 (SIGABRT) means that the process was aborted. This usually means an internal Rosetta error caused by bad inputs, (sometimes) developer error, or (rarely) hardware problems.

I’ve taken the following steps to troubleshoot, but the issue remains:
Verified that the input files are properly formatted and contain valid data within expected ranges.
Checked that the parameters passed to the protocol are valid and that no residue data appears missing or out of bounds.
Confirmed that my system has sufficient memory and resources for the process.

I’ve also checked the ROSETTA_CRASH.log file, but I’m still unsure of how to address the problem. I would greatly appreciate any suggestions or insights on how to resolve this issue.
[ROSETTA_CRASH.log] (https://github.com/user-attachments/files/17205548/ROSETTA_CRASH.log)

**Comment by roccomoretti** [comment]
Unfortunately, the ROSETTA_CRASH.log isn't really giving me much to go on.

If you can, could you recompile Rosetta in debug mode (`mode=debug` on the scons command line, and then run with rosetta_scripts.default.linuxgccdebug instead of rosetta_scripts.default.linuxgccrelease)? That should provide more details about where it's crashing. Also, it may turn on extra error checking which might catch issues earlier.

If you could also post the tracer output (what's printed to the terminal) in addition to the ROSETTA_CRASH.log file, that would also help.
